src/samplers/cpu/mod.rs

```diff
@@ -5,6 +5,8 @@ sampler!(Cpu, "cpu", CPU_SAMPLERS);
 mod stats;
 
 #[cfg(target_os = "linux")]
-mod perf;
-
-mod proc_stat;
+#[path = "."]
+mod linux {
+    mod perf;
+    mod proc_stat;
+}
```

Comment:
This is gonna be confusing for people coming into the code. Why not juts move the perf and proc_state files into a `linux` subfolder?